libs/agno/agno/agent/agent.py

```diff
@@ -1629,11 +1652,14 @@ async def _arun(
             if run_response.metrics:
                 run_response.metrics.stop_timer()
 
-            # 10. Execute post-hooks (after output is generated but before response is returned)
+            # Execute post-hooks (after output is generated but before response is returned)
```

Comment:
```suggestion
            # 10. Execute post-hooks (after output is generated but before response is returned)
```

Not sure why we removed the numbers, ignore if it was conscious :)